src/Badge.js

```diff
@@ -34,6 +34,7 @@ const Badge = (props) => {
     className,
     'badge',
     'bg-' + color,
+    color === "light" && "text-secondary", 
```

Comment:
We should use `text-dark` instead to be more inline with the bootstrap documentation https://getbootstrap.com/docs/5.1/components/badge/#background-colors